Dungeoneer/src/com/badlogic/gdx/utils/TagSet.java

```diff
@@ -0,0 +1,10 @@
+package com.badlogic.gdx.utils;
+
+import java.util.HashSet;
```

Comment:
Be warned, the default Java collections are not always great for performance since they sometimes create extra objects when accessed. LibGdx includes some more game-friendly collections, could this be done with something like the libGdx ArrayMap? https://libgdx.badlogicgames.com/ci/nightlies/docs/api/com/badlogic/gdx/utils/ArrayMap.html

Reply:
I opted in for the `ObjectSet` mainly because of the API compatibility, and also because the Set (as opposed to the Map) interface stores only a set of unique keys. There is no use for the values in the tag system.

Reply:
Thanks! That looks nicer.